### PR TITLE
Display project id (5-letter) in experiment navigation header

### PR DESCRIPTION
**Pull request by wow-such-code**
image
not happy with the "/" - suggestions appreciated

### CODE REVIEW

**Comment by sven1103** [comment]
When navigating to different experiments, the title bar starts to grow:

image

**Comment by sven1103** [comment]
Access page does not show the project ID

image

**Comment by sven1103** [comment]
Ontology lookup does not show the id
image

**Review by sven1103 — DISMISSED** [review]
Dear @wow-such-code , the rendering of the project ID and experiment looks really clean and adds to the overall overview.

There is a bug in the title bar behaviour, when navigating through experiments and in some pages the ID is still missing.

Once this is addressed, I will gladly approve

File: user-interface/src/main/java/life/qbic/datamanager/views/projects/project/ProjectMainLayout.java [hunk 1]
```diff
@@ -81,7 +84,22 @@ public void beforeEnter(BeforeEnterEvent beforeEnterEvent) {
...
+              Icon book = styleIcon(VaadinIcon.NOTEBOOK);
+
+              projectTitle.add(book, projectCode, projectName);
+            });
+  }
+
+  private Icon styleIcon(VaadinIcon vaadinIcon) {
+    Icon icon = vaadinIcon.create();
+    icon.addClassName("primary");
+    icon.setSize("1em");
```
Comment (Steffengreiner) [inline user-interface/src/main/java/life/qbic/datamanager/views/projects/project/ProjectMainLayout.java:101]: NIT: 
Could be replaced with a css classname to make it reusable in other context (e.g. icon.smallest) 
```suggestion
    icon.setSize("1em");
```

**Review by Steffengreiner — APPROVED** [review]
I have a minor nitpick but otherwise good job 👍

**Review by Steffengreiner — APPROVED** [review]
Well done

**Pull request merged**